config/configs/geolix.exs

```diff
@@ -0,0 +1,12 @@
+use Mix.Config
+
+config :geolix,
+  databases: [
+    %{
+      id: :country,
+      adapter: Geolix.Adapter.MMDB2,
+      init: {MaxMindInitializer, :init}
+    }
+  ],
+  banned_countries: System.get_env("BANNED_COUNTRIES"),
+  database_location:  System.get_env("DATABASE_LOCATION")
```

Comment:
Just a comment.. maybe we should qualify this env var for maxmind.. e.g. MMDB_LOCATION ?